music21/metadata/__init__.py

```diff
@@ -1117,21 +1844,54 @@ class RichMetadata(Metadata):
     >>> 'keySignatureFirst' in richMetadata.searchAttributes
     True
     >>> richMetadata.searchAttributes
-    ('actNumber', 'alternativeTitle', 'ambitus', 'associatedWork', 'collectionDesignation',
-     'commission', 'composer', 'copyright', 'countryOfComposition', 'date', 'dedication',
-     'fileFormat', 'fileNumber', 'filePath',
-     'groupTitle', 'keySignatureFirst', 'keySignatures', 'localeOfComposition', 'movementName',
-     'movementNumber', 'noteCount', 'number', 'numberOfParts',
-     'opusNumber', 'parentTitle', 'pitchHighest',
-     'pitchLowest', 'popularTitle', 'quarterLength', 'sceneNumber', 'sourcePath', 'tempoFirst',
-     'tempos', 'textLanguage', 'textOriginalLanguage', 'timeSignatureFirst',
-     'timeSignatures', 'title', 'volume')
+    ('abstract', 'accessRights', 'accompanyingMaterialWriter', 'accrualMethod',
+    'accrualPeriodicity', 'accrualPolicy', 'actNumber', 'actor', 'adapter',
+    'afterwordAuthor', 'alternativeTitle', 'ambitus', 'animator', 'annotator',
+    'architect', 'arranger', 'artist', 'associatedWork', 'attributedComposer',
+    'audience', 'author', 'bibliographicCitation', 'calligrapher', 'cartographer',
```

Comment:
again, far too long now.  Need to be selective.

Reply:
Done.